Sources/Sentry/SentryAppStartTracker.m

```diff
@@ -0,0 +1,181 @@
+#import "SentryAppStartMeasurement.h"
+#import "SentryAppStateManager.h"
+#import "SentryLog.h"
+#import <Foundation/Foundation.h>
+#import <SentryAppStartTracker.h>
+#import <SentryAppState.h>
+#import <SentryClient+Private.h>
+#import <SentryCurrentDateProvider.h>
+#import <SentryDispatchQueueWrapper.h>
+#import <SentryFileManager.h>
+#import <SentryHub.h>
+#import <SentryInternalNotificationNames.h>
+#import <SentryLog.h>
+#import <SentrySDK+Private.h>
+#import <SentrySpan.h>
+#import <SentrySysctl.h>
+
+#if SENTRY_HAS_UIKIT
+#    import <UIKit/UIKit.h>
+#endif
+
+static NSDate *appStart = nil;
```

Comment:
If it's possible this is going to be read by another thread and CPU cache be in the middle if it seeing the value that was set here, we might need to make this `volatile`. If not, best leave without it since once we set data is immutable.

```suggestion
static NSDate volatile *appStart = nil;
```

Reply:
We only set the `appStart` once in the beginning in `load`. This is long before we read `appStart`. Even if it might be a different thread, which it shouldn't because we read in `didBecomeActive` on the main thread, this happens at least 100 ms after we write to the value. I don't think `volatile` is needed here at the moment.